MetabolicAtlas/MetabolicAtlas
# feat: timeline gems visualization

**Related issue(s) and PR(s)**  
This PR closes #949 .



**Type of change**  

 
**List of changes made**  

Add a timeline chart for history of integrated models for the `GEM Repository` page

**Screenshot of the fix**  

<img width="1540" alt="image" src="https://user-images.githubusercontent.com/423498/166683422-185704bb-82c7-4c31-8037-86fa003dfa01.png">
<img width="1529" alt="image" src="https://user-images.githubusercontent.com/423498/166683510-bdd91d59-b457-4e67-80ed-c13a05234017.png">


**Testing**  

- A certain branch in other repos is needed
- A rebuild is needed due to new dependencies
- Relative urls that can be reused both for production and local testing
- Instructions on how to test
  1. This PR requires the work from https://github.com/MetabolicAtlas/data-generation/pull/27. 
  2. Rebuild the project and start it: `build-stack && start-stack`.
  3. Visit the `/gems/repository` page and the timeline chart should appear below the list of integrated models.
  4. Click different versions (circles) and verify that the links work correctly.

**Definition of Done checklist**
